### DISCUSSION TITLE
Unable to get Named Selections using PyMechanical

### DISCUSSION THREAD

**Discussion by Naveen-Kumar-B** [ansys/pymechanical#1012] [Q&A] [answered]
Hi Team, I need help on the NS that are not retrieving in the PyMechanical

**Comment by MeenaBytes** [comment, marked as answer]
Hi Naveen,

You can add empty named selection key, it should work.

geometry_import_preferences.NamedSelectionKey = ""

**Reply by Naveen-Kumar-B** [reply]
Thanks Rajesh its working fine
